src/connection.c

```diff
@@ -66,9 +66,9 @@ Connection *ConnCreate(RedisRaftCtx *rr, void *privdata,
 
     int timeout_millis = rr->config.connection_timeout;
 
-    conn->timeout = (struct timeval) {
-            .tv_sec = timeout_millis / 1000,
-            .tv_usec = (timeout_millis % 1000) * 1000
+    conn->timeout = (struct timeval){
+        .tv_sec = timeout_millis / 1000,
+        .tv_usec = (timeout_millis % 1000) * 1000,
```

Comment:
See the trailing comma. Without it, clang-format will not format this way. We need to add it when required for arrays and designated struct initializers.